cmd/cortex/main_test.go

```diff
@@ -0,0 +1,162 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"io"
+	"io/ioutil"
+	"os"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestName(t *testing.T) {
```

Comment:
Did you mean `TestMain`?

Reply:
That was default name autocompleted by Golang. I forgot to change it :) Fixed now.